libminifi/src/utils/StringUtils.cpp

```diff
@@ -95,21 +95,23 @@ bool StringUtils::StringToFloat(std::string input, float &output, FailurePolicy
 }
 
 std::string StringUtils::replaceEnvironmentVariables(std::string& original_string) {
-  int32_t beg_seq = 0;
-  int32_t end_seq = 0;
+  size_t beg_seq = 0;
```

Comment:
should we use `std::string::size_type`?

Reply:
I think we can assume that `string::size_type` will always be `size_t`, but you are right, `string::size_type` is more correct, so I changed it.